src/components/ExpandedBankTransactionRow/ExpandedBankTransactionRow.tsx

```diff
@@ -441,14 +477,101 @@ export const ExpandedBankTransactionRow = forwardRef<SaveHandle, Props>(
 
     useEffect(() => {
       const loadDocumentsAndMetadata = async () => {
-        if (showDescriptions) await fetchMetadata()
-        if (showReceiptUploads) await fetchDocuments()
+        if (showDescriptions && bankTransaction?.metadata.memo)
+          setMemoText(bankTransaction?.metadata.memo)
```

Comment:
Made this change so it doesn't separately call the metadata endpoint for each transaction on initial load, instead using the metadata field on the bank transaction object.